torch/_inductor/fx_passes/freezing_patterns.py

```diff
@@ -24,15 +27,31 @@
 binary_folding_pass = PatternMatcherPass()
 
 
-def freezing_passes(gm: torch.fx.GraphModule):
+def freezing_passes(gm: torch.fx.GraphModule, aot_example_inputs):
     """
     Passes that are applied to the graph to freeze pass.
     """
 
+    from ..freezing import constant_fold
+
     lazy_init()
-    binary_folding_pass.apply(gm.graph)
-    for patterns in pass_patterns:
-        patterns.apply(gm.graph)
+    # We need a few rounds of binary folding to get rid of all the
+    # unnecessary nodes, but may need a good method to chose the rounds number.
+    # works like: conv+binary+binary.
+    binary_folding = 0
```

Comment:
im not 100% sure the counters get reset so i would initialize `binary_folding = counters["inductor"]["binary_folding"]`

Reply:
changed.